packages/api/prisma/schema.prisma

```diff
@@ -220,10 +220,67 @@ model Comment {
 
   user User? @relation(fields: [userID], references: [id])
 
+  ratings CommentRating[]
+
   @@index([createdAt])
   @@map("comments")
 }
 
+model CommentRatingSystem {
```

Comment:
For now, the UI is not consistent with the provided data structure here. There should actually be a list view with all rating systems, which a user could edit one by one.
Do we just skip it?

Reply:
We decided that in the Miro

![image](https://user-images.githubusercontent.com/1131901/190107381-5e105a62-f5b1-4846-a695-d5d727219432.png)